src/implementations/vanilla/components/featured-item/_featured-item.scss

```diff
@@ -112,6 +110,21 @@ $_extended-padding-bottom: null !default;
     }
   }
 
+  .ecl-featured-item__container--row-reverse {
```

Comment:
I'm not a big fan of this long class name...
Maybe we could use a "shortcut", and name it ecl-featured-item__container--reverse, or even ecl-featured-item__container--right; that would not be totally accurate, as it is the picture alignment, but that would be somehow consistent with what we have on the content item